pkgs/development/compilers/heptagon/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, makeWrapper
+, ocamlPackages
+}:
+
+with ocamlPackages;
+
```

Comment:
I think it would be [better](https://nix.dev/anti-patterns/language#with-attrset-expression) to reduce the scope of this.
```suggestion
```

Reply:
Done.